doc/source/examples/multiphysics/rayleigh-taylor-instability/rayleigh-taylor-instability.rst

```diff
@@ -0,0 +1,252 @@
+        
+        subsection VOF
+                set Function expression = if (y>(0.5 + 0.1 * 0.25 * cos(2 *3.1415 * x / 0.25)) , 1, 0)
+        end
+    end
+
+
+The ``initial refinement`` of the mesh is equal to 7, but we use a mesh adaptation to coarsen the mesh in cells far from the interface to improve the computation performance. Here, we choose ``phase`` as the ``refinement variable`` and 5 as the ``min refinement level``. 
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # Mesh Adaptation
+    #---------------------------------------------------
+    subsection mesh adaptation
+      set type                    	= kelly
+      set variable                	= phase
+      set fraction type           	= fraction
+      set max refinement level    	= 7
+      set min refinement level    	= 5
+      set frequency               	= 1
+      set fraction refinement     	= 0.9
+      set fraction coarsening     	= 0.001
+    end
+
+
+In the ``VOF`` subsection, we enable ``interface sharpening`` to reconstruct the interface and keep it sharp during the simulation. Note that here we use the ``constant`` method for interface sharpening. We can also use ``adaptative`` type for interface sharpening, but the ``mass conservation`` results show that choosing a ``constant`` method does not affect the mass conservation significantly.
+
+
+.. code-block:: text
+
+    #---------------------------------------------------
+    # VOF
+    #---------------------------------------------------
+    subsection VOF
+      subsection interface sharpening
+        set enable      		= true
+        set threshold		= 0.5
+        set interface sharpness	= 1.5
+        set frequency		= 25
+        set type        		= constant
+      end
+     
+      subsection mass conservation
+        set monitoring          	= true
+        set monitored fluid     	= fluid 1
+        set tolerance           	= 1e-6
+        set verbosity           	= quiet
+      end
+    end
+
+
+---------------------------
+Running the simulation
+---------------------------
+
+Call the gls_navier_stokes_2d by invoking:  
+
+``mpirun -np 8 gls_navier_stokes_2d rayleigh_taylor_instability.prm``
+
+
+to run the simulations using eight CPU cores. Feel free to use more.
+
+.. warning:: 
+    Make sure to compile lethe in `Release` mode and 
+    run in parallel using mpirun. This simulation takes
+    :math:`\approx` 10 minutes on 8 processes.
+
+
+-------
+Results
+-------
+
+The following animation shows the results of this simulation:
+
+.. raw:: html
+
+    <iframe width="560" height="315" src="https://www.youtube.com/embed/hZwbFob_Jj4" frameborder="0" allowfullscreen></iframe>
+
+
+In the following figure, we compare the simulation results with that of Garoosi and Hooman [2].
+
+
+.. image:: images/comparison.png
+    :alt: Schematic
+    :align: center
+    :width: 400
+
+
+The following figure shows the mass of fluid 1 throughout the simulation.
```

Comment:
Maybe you can choose a better y axis scale? It's a bit too zoomed out.

Reply:
Done.